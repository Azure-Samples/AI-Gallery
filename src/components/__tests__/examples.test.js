import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import Examples from './../examples';


test('renders a snapshot of Examples', () => {
    const tree = renderer.create(<Examples/>).toJSON();
    expect(tree).toMatchSnapshot();
})