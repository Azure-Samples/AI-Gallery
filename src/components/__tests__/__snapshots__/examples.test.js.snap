// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of Examples 1`] = `
<div>
  <div
    class="menu-wrapper"
    id="example-wrapper"
  >
    <div>
      <ul
        className="menu examples"
        id="examples-menu"
      >
        <li
          className="examplesitem item"
        >
          <div>
            <div
              className="github-card repo-card"
            >
              <div
                className={
                  Array [
                    "header",
                  ]
                }
              >
                <a
                  className="avatar"
                  href="https://github.com/Azure"
                  target="_top"
                >
                  <img
                    alt="Azure"
                    src="https://avatars0.githubusercontent.com/u/6844498?v=4"
                  />
                </a>
                <h1>
                  <a
                    className="wordwrap"
                    href="https://www.github.com/Azure/MachineLearningSamples-PredictiveMaintenance"
                    target="_top"
                  >
                    MachineLearningSamples-PredictiveMaintenance
                  </a>
                </h1>
              </div>
              <div
                className="content"
              >
                <p>
                  MachineLearningSamples-PredictiveMaitenance
                </p>
              </div>
              <ul
                className="status"
              >
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-PredictiveMaintenance/network"
                    target="_top"
                  >
                    <strong>
                      30
                    </strong>
                    Forks
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-PredictiveMaintenance/stargazers"
                    target="_top"
                  >
                    <strong>
                      34
                    </strong>
                    Stars
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </li>
        <li
          className="examplesitem item"
        >
          <div>
            <div
              className="github-card repo-card"
            >
              <div
                className={
                  Array [
                    "header",
                  ]
                }
              >
                <a
                  className="avatar"
                  href="https://github.com/Azure"
                  target="_top"
                >
                  <img
                    alt="Azure"
                    src="https://avatars0.githubusercontent.com/u/6844498?v=4"
                  />
                </a>
                <h1>
                  <a
                    className="wordwrap"
                    href="https://www.github.com/Azure/MachineLearningSamples-SentimentAnalysis"
                    target="_top"
                  >
                    MachineLearningSamples-SentimentAnalysis
                  </a>
                </h1>
              </div>
              <div
                className="content"
              >
                <p>
                  MachineLearningSamples-SentimentAnalysis
                </p>
              </div>
              <ul
                className="status"
              >
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-SentimentAnalysis/network"
                    target="_top"
                  >
                    <strong>
                      19
                    </strong>
                    Forks
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-SentimentAnalysis/stargazers"
                    target="_top"
                  >
                    <strong>
                      28
                    </strong>
                    Stars
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </li>
        <li
          className="examplesitem item"
        >
          <div>
            <div
              className="github-card repo-card"
            >
              <div
                className={
                  Array [
                    "header",
                  ]
                }
              >
                <a
                  className="avatar"
                  href="https://github.com/Azure"
                  target="_top"
                >
                  <img
                    alt="Azure"
                    src="https://avatars0.githubusercontent.com/u/6844498?v=4"
                  />
                </a>
                <h1>
                  <a
                    className="wordwrap"
                    href="https://www.github.com/Azure/MachineLearningSamples-ChurnPrediction"
                    target="_top"
                  >
                    MachineLearningSamples-ChurnPrediction
                  </a>
                </h1>
              </div>
              <div
                className="content"
              >
                <p>
                  MachineLearningSamples-ChurnPrediction
                </p>
              </div>
              <ul
                className="status"
              >
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-ChurnPrediction/network"
                    target="_top"
                  >
                    <strong>
                      24
                    </strong>
                    Forks
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-ChurnPrediction/stargazers"
                    target="_top"
                  >
                    <strong>
                      20
                    </strong>
                    Stars
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </li>
        <li
          className="examplesitem item"
        >
          <div>
            <div
              className="github-card repo-card"
            >
              <div
                className={
                  Array [
                    "header",
                  ]
                }
              >
                <a
                  className="avatar"
                  href="https://github.com/Azure"
                  target="_top"
                >
                  <img
                    alt="Azure"
                    src="https://avatars0.githubusercontent.com/u/6844498?v=4"
                  />
                </a>
                <h1>
                  <a
                    className="wordwrap"
                    href="https://www.github.com/Azure/MachineLearningSamples-BiomedicalEntityExtraction"
                    target="_top"
                  >
                    MachineLearningSamples-BiomedicalEntityExtraction
                  </a>
                </h1>
              </div>
              <div
                className="content"
              >
                <p>
                  MachineLearningSamples-BiomedicalEntityExtraction
                </p>
              </div>
              <ul
                className="status"
              >
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-BiomedicalEntityExtraction/network"
                    target="_top"
                  >
                    <strong>
                      18
                    </strong>
                    Forks
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-BiomedicalEntityExtraction/stargazers"
                    target="_top"
                  >
                    <strong>
                      32
                    </strong>
                    Stars
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </li>
        <li
          className="examplesitem item"
        >
          <div>
            <div
              className="github-card repo-card"
            >
              <div
                className={
                  Array [
                    "header",
                  ]
                }
              >
                <a
                  className="avatar"
                  href="https://github.com/Azure"
                  target="_top"
                >
                  <img
                    alt="Azure"
                    src="https://avatars0.githubusercontent.com/u/6844498?v=4"
                  />
                </a>
                <h1>
                  <a
                    className="wordwrap"
                    href="https://www.github.com/Azure/MachineLearningSamples-QnAMatching"
                    target="_top"
                  >
                    MachineLearningSamples-QnAMatching
                  </a>
                </h1>
              </div>
              <div
                className="content"
              >
                <p>
                  MachineLearningSamples-QnAMatching
                </p>
              </div>
              <ul
                className="status"
              >
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-QnAMatching/network"
                    target="_top"
                  >
                    <strong>
                      12
                    </strong>
                    Forks
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-QnAMatching/stargazers"
                    target="_top"
                  >
                    <strong>
                      13
                    </strong>
                    Stars
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </li>
        <li
          className="examplesitem item"
        >
          <div>
            <div
              className="github-card repo-card"
            >
              <div
                className={
                  Array [
                    "header",
                  ]
                }
              >
                <a
                  className="avatar"
                  href="https://github.com/Azure"
                  target="_top"
                >
                  <img
                    alt="Azure"
                    src="https://avatars0.githubusercontent.com/u/6844498?v=4"
                  />
                </a>
                <h1>
                  <a
                    className="wordwrap"
                    href="https://www.github.com/Azure/MachineLearningSamples-AerialImageClassification"
                    target="_top"
                  >
                    MachineLearningSamples-AerialImageClassification
                  </a>
                </h1>
              </div>
              <div
                className="content"
              >
                <p>
                  MachineLearningSamples-AerialImageClassification 
                </p>
              </div>
              <ul
                className="status"
              >
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-AerialImageClassification/network"
                    target="_top"
                  >
                    <strong>
                      5
                    </strong>
                    Forks
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.github.com/Azure/MachineLearningSamples-AerialImageClassification/stargazers"
                    target="_top"
                  >
                    <strong>
                      11
                    </strong>
                    Stars
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </li>
      </ul>
      <div
        class="paddles"
      >
        <button
          class="left-paddle paddle hidden"
          id="examples-left"
          onClick={[Function]}
        >
          &lt;
        </button>
        <button
          class="right-paddle paddle"
          id="examples-right"
          onClick={[Function]}
        >
          &gt;
        </button>
      </div>
    </div>
  </div>
</div>
`;
